meta/autoid/autoid.go

```diff
@@ -375,7 +394,11 @@ func NewSequenceAllocator(store kv.Storage, dbID int64, info *model.SequenceInfo
 func NewAllocatorsFromTblInfo(store kv.Storage, schemaID int64, tblInfo *model.TableInfo) Allocators {
 	var allocs []Allocator
 	dbID := tblInfo.GetDBID(schemaID)
-	allocs = append(allocs, NewAllocator(store, dbID, tblInfo.IsAutoIncColUnsigned(), RowIDAllocType))
+	if tblInfo.AutoIdCache > 0 {
```

Comment:
Do there a test for the case `AutoIdCache == -10`?

Reply:
this element should be `GE` to 0 (default), otherwise, create table statement will parser error.